src/storage/README.md

```diff
@@ -47,9 +51,30 @@ version. These features will be rolled out in future updates._
 ## Reading an encrypted file
 
 ```JavaScript
- blockstack.getFile("/message.txt", true)
+ let options = { 
+   decrypt: true 
+ }
+
+ blockstack.getFile("/message.txt", options)
  .then((fileContents) => {
     // get & decrypt the contents of the file /message.txt
     assert(fileContents === "Secret hello!")
  });
 ```
+
+## Reading another user's unencrypted file
+In order for files to be publicly readable, the app must request 
+the `publish_data` scope during authentication.
+
+```JavaScript
+ let options = { 
+   user: 'ryan.id', // the Blockstack ID of the user for which to lookup the file
+   app: 'BlockstackApp.com' // origin of the app this file is stored for
```

Comment:
This example should be an origin: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin#Syntax